src/Test/TestState.php

```diff
@@ -91,9 +112,17 @@ public static function bootFoundry(?Configuration $configuration = null): void
             $configuration->disableDefaultProxyAutoRefresh();
         }
 
+        self::bootFoundry($configuration);
+    }
+
+    public static function bootFoundry(Configuration $configuration): void
```

Comment:
I was thinking to drop this method because it just call another static method, but maybe it could be useful to keep it to centralize "boot" step, if maybe we got something else than Factory to boot

Reply:
Thinking we should mark everything in here except `configure()` as `@internal`/`@deprecated`, wdyt?

Reply:
absolutely!